pom.xml

```diff
@@ -149,7 +149,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
     <!-- ProtocolBuffer version, used to verify the protoc version and -->
     <!-- define the protobuf JAR version                               -->
-    <protobuf.version>2.5.0</protobuf.version>
+    <protobuf.version>3.10.0</protobuf.version>
```

Comment:
Can we rename this to something like hadoop-protobuf-compile version ?

Reply:
Also can this be the same as the "<protobuf-compile.version>3.5.0</protobuf-compile.version>"

Reply:
In #300 `protobuf-compile.version` has been disappeared and we have only `datanode.protobuf-compile.version`. I reorganized this setting to be more similar:

```
   <!-- Maven protoc compiler -->
    <protobuf-maven-plugin.version>0.5.1</protobuf-maven-plugin.version>
    <hadooprpc.protobuf-compile.version>3.10.0</hadooprpc.protobuf-compile.version>
    <datanode.protobuf-compile.version>3.10.0</datanode.protobuf-compile.version>
 ```

But let me know if you have any better idea...

Reply:
This sounds perfect to me.